theme/int/views/authentication/home.hbs

```diff
@@ -29,15 +29,143 @@
         <!-- GHOST (will be replaced by js) -->
         <div class="hidden" id="SC_THEME">int</div>
         <div class="int-blog">
+            <div class="content" style="opacity: 1; transition: opacity 0.7s ease-in;">
+                <div class="container">
+                    <div class="row section-landing flex-container">
+                        <div class="col-sm-6">
+                            <div class="pull-left">
+                                <h6 class="font-weight-bold">HPI Schul-Cloud International</h6>
+                                <h1 class="mb-3">Lernen global vernetzt<br>- digital unterstützt</h1>
```

Comment:
Not sure whether Ghost CMS would be able to process this. I also didn't write this text myself, it's just the modified default which is now used in production. If it works with Ghost I surely can use the translation engine (would be some work and should maybe be another ticket.)

Reply:
I'm pretty sure it will work.
It is a *.hbs file. I don't know exactly what Ghost CMS does with it, but I'm sure it will be processed by handlebars. Otherwise all the `{{#content}}` wouldn't work either.